### PR TITLE
re-grouping and plitting out of sql test files

### PR DESCRIPTION
**Pull request by navonod**
I've tried to split these into the logical areas of the system - please feel free to make suggestions to move anything that you feel is a better fit elsewhere.

I removed the priming files completely, as we don't plan on keeping many of these. The strategy should be to prime the test from code, where it's in plain view and easy to access.

### CODE REVIEW

File: scripts/v6/asset-admin.sql [hunk 1]
```diff
@@ -0,0 +1,115 @@
+-- Various tables used to maintain metadata, or manage components within th eWIP or AR.
```
Comment (frankvanrietimqs) [inline scripts/v6/asset-admin.sql]: This should be split up:
- refcounter,finyear can stay here
-  changeiconfin is part of the asset register (it is the asset register change log)  
-  I think project should be in a file called v6Wip or something similar.

Please split schema and population. As I have previously mentioned before the real value is in sharing the schema - the data is likely to change depending on the test being run.

File: scripts/v6/asset-ref.sql [hunk 2]
```diff
@@ -0,0 +1,83 @@
+-- Reference tables for constant values that don't change much and are associated to AR fields via lookups
```
Comment (frankvanrietimqs) [inline scripts/v6/asset-ref.sql]: I would move the schema into the asset register - it is inextricably linked

File: scripts/v6/views/input-forms.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	[Rec_Action],[Carrying_Value],[Value_in_Use],[Fair_Value_Less],[Recoverable_Amt],[Police_Report_Nr],[Take_on_Date],[Reason_Code],
+	[form_timestamp] [timestamp]
+FROM
+	AssetFinFormInput
+JOIN
+	AssetFinFormRef on AssetFinFormInput.Form_Reference = AssetFinFormRef.Form_Reference
+JOIN
+	AssetFinForm on AssetFinForm.Form_Nr = AssetFinFormRef.Form_Nr
+JOIN
+	AssetFinFormState on AssetFinFormState.Form_Level = AssetFinFormRef.Form_Level AND AssetFinFormState.Workflow = 'GENERIC';
```
Comment (nzali) [inline scripts/v6/views/input-forms.sql]: It seems like AssetFinFormState.Workflow changed. there's no longer 'GENERIC'. I don't know what it should be. @venterbg69 would be able to assist?

File: scripts/v6/asset-fin-forms.sql [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+	NULL, NULL, NULL, N'TestProject001', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 2013, NULL, NULL, NULL, CAST(25000.00 AS Numeric(18, 2)),
+	NULL, NULL, NULL, NULL, 15, NULL, CAST(15.00 AS Numeric(18, 2)), 0, CAST(0.00 AS Numeric(18, 2)), N'ZJW1', NULL, CAST(0.00 AS Numeric(18, 2)), NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
+	NULL, NULL, NULL, NULL, NULL, NULL, CAST(377300.00 AS Numeric(18, 2)), NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, CAST(377300.00 AS Numeric(18, 2)), NULL,
+	NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, N'comp1', NULL, NULL, NULL, N'78987', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 78987, 78987, 78987, 1, 99092, NULL, NULL, NULL, NULL, NULL,
+	NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, CAST(0.00 AS Numeric(18, 2)), CAST(0.00 AS Numeric(18, 2)), CAST(0.00 AS Numeric(18, 2)), NULL, NULL, NULL, NULL, NULL, NULL,
+	N'240612638294574528', CAST(0.00 AS Numeric(18, 2)), NULL, NULL, NULL, NULL, NULL, NULL, NULL, 2015
+);
+INSERT [dbo].[AssetFinFormInput](
```
Comment (nzali) [inline scripts/v6/asset-fin-forms.sql]: We could also refactor these insert statements....get rid of NULL values that we don't really need them.

**Review by nzali — CHANGES_REQUESTED** [review]

File: scripts/v6/views/input-forms.sql (hunk 3 above)
Comment (navonod) [reply scripts/v6/views/input-forms.sql]: We can look at the tests using this view, and change them id required, though I don't think it's essential right now, as they aren't specifically testing the workflow state itself.

As for the rollup stored proc tests, technically the new values would determine how they determine whether or not to select SCOAJournal entries by CommittedToRegister=1 or CommittedToRegister=0, but the tests express this explicitly as params passed to the stored procs.

File: scripts/v6/asset-fin-forms.sql (hunk 4 above)
Comment (navonod) [reply scripts/v6/asset-fin-forms.sql]: We're going to be redoing the insert scripts with CSV files in any case, so this concern will be handled then.

**Pull request merged**